### PR TITLE
Introducing unit tests

### PR DESCRIPTION
**Pull request by Art4**
refs #4.

I've created all files needed for automate testing with Travis and code coverage with Coveralls. See:
- Travis: https://travis-ci.org/Art4/lassi
- Coveralls: https://coveralls.io/github/Art4/lassi

Like in #6 described I rename the folder because Unix is case-sensitive. I also moved the namespace `Lassi` into a separate folder. This keeps it easier for testing and code coverage.

I also corrected the dependendy from PHP 5.4 to PHP 5.5.9 because illuminate/database requires it at a minimum.

I write also a first test for `Lassi::getInstance()`. The Lassi code isn't easy to test because of the dependencies inside the classes. If you would allow to set all dependencies from outside (inversion of control) this would make it possible to mock specific classes and write real unit tests.

However this PR is ready for being merged. It allows to write more tests in the future. Let me know what you think

### CODE REVIEW

**Comment by jabranr** [comment]
Incredible work indeed. I acknowledge these issues that you have mentioned. I was planning to rewrite it to make it TDD compatible but just didn't get enough time. You might get some traces of it in other branches though. I think the complex code is because it never was based on TDD approach.

Let me go through all of these updates introduced in this PR and I'll come back – with feedback if any.

File: app/Util.php [hunk 1]
```diff
@@ -40,6 +40,13 @@ public static function setEnvVariables($root = '/') {
 				die($e->getMessage());
 			}
 		}
+		else if ( file_exists($root . '/.test.env') && is_readable($root . '/.test.env') ) {
+			try {
```
Comment (jabranr) [inline app/Util.php:44]: I think I will update this file to remove `try catch` block as `file_get_contents` won't throw an exception.

**Review by jabranr — APPROVED** [review]

File: app/Util.php (hunk 1 above)
Comment (Art4) [reply app/Util.php:44]: You're right.

**Comment by jabranr** [comment]
Resolves issue #4

**Pull request merged**